elastic/kibana
# [Security Solutions] Hide User flyout managed data section behind flag

## Summary
Hide User flyout managed data section behind an experimental flag (`newUserDetailsFlyoutManagedUser`)

<img width="300" src="https://github.com/elastic/kibana/assets/1490444/dd409cad-fcfe-4f53-a122-e6b7c293de4e"/>

### How to test it?
* Open security solutions app and ingest data
* Create alerts
* Enable risk engine
* Go to the Alerts page and click on the user name cell
* The sections should **not** show
* Enable the flag and follow the same steps to see the section


### Checklist
